### DISCUSSION TITLE
Minor issue with allotment package

### DISCUSSION THREAD

**Discussion by johnwalley** [AykutSarac/jsoncrack.com#31] [General]
Hi @AykutSarac. I'm the maintainer of the allotment package (thanks for using it!). I think it has a bug whereby the sash isn't properly centred on the separator between panes. I need to investigate what's going and I've created an issue to track progress: https://github.com/johnwalley/allotment/issues/269.

It's a minor thing but I just wanted to let you know I'm aware of the issue.

**Comment by AykutSarac** [comment]
Hello @johnwalley! Thanks for the heads up, I will be looking forward hearing from you. Love the package :)

**Comment by johnwalley** [comment]
Oh, this is funny. Allotment is based on the vs code split view implementation. You're also using the Monaco editor which, I assume, is literally using the vs code split view implementation. Both libraries share some of the same css variable names. In particular, `--sash-size` is used by both libraries. I'll think harder about how to resolve it, i.e. it's an allotment issue.

In the meantime, there is an exported function `setSashSize` which if you call it with the same value as used by the Monaco editor should make things look correct, i.e. `setSashSize(4)`.
